measure_cocoapod_size.py

```diff
@@ -166,18 +166,26 @@ def GetPodSizeImpact(parsed_args):
     raise ValueError("could not decode JSON value %s: %s" % (parsed_args.cocoapods_source_config.name, e))
   base_project = tempfile.mkdtemp()
   target_project = tempfile.mkdtemp()
+  target_dir = os.path.join(target_project, sample_app_dir)
   CopyProject(sample_app_dir, base_project)
   CopyProject(sample_app_dir, target_project)
 
   target_project = InstallPods(cocoapods,
-                               os.path.join(target_project, sample_app_dir),
+                               target_dir,
                                spec_repos, sample_app_name, parsed_args.mode,
                                pod_sources)
   source_project = os.path.join(base_project,
                                 '{}/{}.xcodeproj'.format(sample_app_dir, sample_app_name))
 
   source_size, target_size = GenerateSizeDifference(
       source_project, sample_app_name, target_project, sample_app_name, parsed_args.build_timeout)
+  if parsed_args.json:
+    # Transfer Podfile to JSON format.
+    podfile, _ = shell('pod ipc podfile-json {}/Podfile'.format(target_dir), capture_output=True)
```

Comment:
Question, can `pod ipc podfile-json` return any stderr?  I see we ignore it when we deconstruct the `shell` function's return. Not sure if we need to account for if `podfile` is `None` before proceeding..

Reply:
Error from the Shell will still be caught, but `capture_output` will set PIPE for both `stdout` and `stderr`, so `stderr` will not be directed to console. Set only `stdout` to PIPE due to the scenario.